### ISSUE TITLE
Note in Second Voice Misaligned with First Voice

### ISSUE THREAD

**Issue by dnlfrst** [opensheetmusicdisplay/opensheetmusicdisplay#902]
I have only encountered the following issue in the music sheet [_Minuet in G Major_](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/files/5377664/Minuet_in_G_Major.musicxml.zip), hence I will use that as the guiding example.

In the measures 30 to 32 (inclusive), the second voice in both staves introduces a dotted quarter note that accompanies the second group of eighth notes. The source (MusicXML) proofs that:

<details>
<summary>MusicXML</summary>

```xml
<measure number="30" width="270.67">
    <print new-page="yes">
        <system-layout>
            <system-margins>
                <left-margin>75.84</left-margin>
                <right-margin>0.00</right-margin>
            </system-margins>
            <top-system-distance>70.00</top-system-distance>
        </system-layout>
        <staff-layout number="2">
            <staff-distance>85.00</staff-distance>
        </staff-layout>
    </print>
    <direction placement="below">
        <direction-type>
            <wedge type="diminuendo" number="1" default-y="-55.40" relative-x="-2.56"/>
        </direction-type>
        <staff>1</staff>
    </direction>
    <note default-x="93.01" default-y="-15.00">
        <pitch>
            <step>C</step>
            <octave>5</octave>
        </pitch>
        <duration>2</duration>
        <voice>1</voice>
        <type>eighth</type>
        <stem>up</stem>
        <staff>1</staff>
        <beam number="1">begin</beam>
    </note>
    <note default-x="123.53" default-y="-25.00">
        <pitch>
            <step>A</step>
            <octave>4</octave>
        </pitch>
        <duration>2</duration>
        <voice>1</voice>
        <type>eighth</type>
        <accidental>natural</accidental>
        <stem>up</stem>
        <staff>1</staff>
        <beam number="1">continue</beam>
    </note>
    <direction placement="below">
        <direction-type>
            <wedge type="stop" number="1" relative-x="3.44"/>
        </direction-type>
        <staff>1</staff>
    </direction>
    <note default-x="149.77" default-y="-35.00">
        <pitch>
            <step>F</step>
            <alter>1</alter>
            <octave>4</octave>
        </pitch>
        <duration>2</duration>
        <voice>1</voice>
        <type>eighth</type>
        <stem>up</stem>
        <staff>1</staff>
        <beam number="1">end</beam>
        <notations>
            <slur type="stop" number="1"/>
        </notations>
    </note>
    <direction placement="below">
        <direction-type>
            <dynamics default-x="3.29" default-y="-80.00" relative-x="-12.70" relative-y="8.40">
                <mf/>
            </dynamics>
        </direction-type>
        <staff>1</staff>
        <sound dynamics="88.89"/>
    </direction>
    <note default-x="176.00" default-y="-10.00">
        <pitch>
            <step>D</step>
            <octave>5</octave>
        </pitch>
        <duration>2</duration>
        <voice>1</voice>
        <type>eighth</type>
        <stem>down</stem>
        <staff>1</staff>
        <beam number="1">begin</beam>
        <notations>
            <slur type="start" number="1"/>
        </notations>
    </note>
    <direction placement="below">
        <direction-type>
            <wedge type="crescendo" number="1" default-y="-57.53" relative-x="-4.07"/>
        </direction-type>
        <staff>1</staff>
    </direction>
    <note default-x="216.61" default-y="-15.00">
        <pitch>
            <step>C</step>
            <alter>1</alter>
            <octave>5</octave>
        </pitch>
        <duration>2</duration>
        <voice>1</voice>
        <type>eighth</type>
        <accidental>sharp</accidental>
        <stem>down</stem>
        <staff>1</staff>
        <beam number="1">continue</beam>
    </note>
    <direction placement="below">
        <direction-type>
            <wedge type="stop" number="1" relative-x="11.93"/>
        </direction-type>
        <staff>1</staff>
    </direction>
    <note default-x="242.84" default-y="-10.00">
        <pitch>
            <step>D</step>
            <octave>5</octave>
        </pitch>
        <duration>2</duration>
        <voice>1</voice>
        <type>eighth</type>
        <stem>down</stem>
        <staff>1</staff>
        <beam number="1">end</beam>
    </note>
    <backup>
        <duration>12</duration>
    </backup>
    <note default-x="68.67" default-y="-145.00">
        <grace slash="yes"/>
        <pitch>
            <step>D</step>
            <octave>3</octave>
        </pitch>
        <voice>5</voice>
        <type>eighth</type>
        <stem>up</stem>
        <staff>2</staff>
        <notations>
            <slur type="start" number="3"/>
        </notations>
    </note>
    <note default-x="93.01" default-y="-105.00">
        <pitch>
            <step>E</step>
            <octave>4</octave>
        </pitch>
        <duration>2</duration>
        <voice>5</voice>
        <type>eighth</type>
        <stem>down</stem>
        <staff>2</staff>
        <beam number="1">begin</beam>
        <notations>
            <slur type="stop" number="3"/>
        </notations>
    </note>
    <note default-x="123.53" default-y="-115.00">
        <pitch>
            <step>C</step>
            <octave>4</octave>
        </pitch>
        <duration>2</duration>
        <voice>5</voice>
        <type>eighth</type>
        <accidental>natural</accidental>
        <stem>down</stem>
        <staff>2</staff>
        <beam number="1">continue</beam>
    </note>
    <note default-x="149.77" default-y="-125.00">
        <pitch>
            <step>A</step>
            <octave>3</octave>
        </pitch>
        <duration>2</duration>
        <voice>5</voice>
        <type>eighth</type>
        <stem>down</stem>
        <staff>2</staff>
        <beam number="1">end</beam>
        <notations>
            <slur type="stop" number="2"/>
        </notations>
    </note>
    <note default-x="176.00" default-y="-120.00">
        <pitch>
            <step>B</step>
            <octave>3</octave>
        </pitch>
        <duration>2</duration>
        <voice>5</voice>
        <type>eighth</type>
        <stem>down</stem>
        <staff>2</staff>
        <beam number="1">begin</beam>
        <notations>
            <slur type="start" number="2"/>
        </notations>
    </note>
    <note default-x="216.61" default-y="-125.00">
        <pitch>
            <step>A</step>
            <alter>1</alter>
            <octave>3</octave>
        </pitch>
        <duration>2</duration>
        <voice>5</voice>
        <type>eighth</type>
        <accidental>sharp</accidental>
        <stem>down</stem>
        <staff>2</staff>
        <beam number="1">continue</beam>
    </note>
    <note default-x="242.84" default-y="-120.00">
        <pitch>
            <step>B</step>
            <octave>3</octave>
        </pitch>
        <duration>2</duration>
        <voice>5</voice>
        <type>eighth</type>
        <stem>down</stem>
        <staff>2</staff>
        <beam number="1">end</beam>
    </note>
    <backup>
        <duration>6</duration>
    </backup>
    <note default-x="176.00" default-y="-110.00">
        <pitch>
            <step>D</step>
            <octave>4</octave>
        </pitch>
        <duration>6</duration>
        <voice>6</voice>
        <type>quarter</type>
        <dot/>
        <stem>up</stem>
        <staff>2</staff>
    </note>
</measure>
<measure number="31" width="207.20">
    <direction placement="below">
        <direction-type>
            <wedge type="diminuendo" number="1" default-y="-64.40" relative-x="-4.89"/>
        </direction-type>
        <staff>1</staff>
    </direction>
    <note default-x="12.00" default-y="-5.00">
        <pitch>
            <step>E</step>
            <octave>5</octave>
        </pitch>
        <duration>2</duration>
        <voice>1</voice>
        <type>eighth</type>
        <stem>down</stem>
        <staff>1</staff>
        <beam number="1">begin</beam>
    </note>
    <note default-x="50.06" default-y="-15.00">
        <pitch>
            <step>C</step>
            <octave>5</octave>
        </pitch>
        <duration>2</duration>
        <voice>1</voice>
        <type>eighth</type>
        <accidental>natural</accidental>
        <stem>down</stem>
        <staff>1</staff>
        <beam number="1">continue</beam>
    </note>
    <direction placement="below">
        <direction-type>
            <wedge type="stop" number="1" relative-x="4.11"/>
        </direction-type>
        <staff>1</staff>
    </direction>
    <note default-x="78.79" default-y="-25.00">
        <pitch>
            <step>A</step>
            <octave>4</octave>
        </pitch>
        <duration>2</duration>
        <voice>1</voice>
        <type>eighth</type>
        <stem>down</stem>
        <staff>1</staff>
        <beam number="1">end</beam>
        <notations>
            <slur type="stop" number="1"/>
        </notations>
    </note>
    <direction placement="below">
        <direction-type>
            <wedge type="crescendo" number="1" default-y="-71.53" relative-x="-3.36"/>
        </direction-type>
        <staff>1</staff>
    </direction>
    <note default-x="107.53" default-y="-20.00">
        <pitch>
            <step>B</step>
            <octave>4</octave>
        </pitch>
        <duration>2</duration>
        <voice>1</voice>
        <type>eighth</type>
        <stem>up</stem>
        <staff>1</staff>
        <beam number="1">begin</beam>
        <notations>
            <slur type="start" number="1"/>
        </notations>
    </note>
    <note default-x="148.14" default-y="-25.00">
        <pitch>
            <step>A</step>
            <alter>1</alter>
            <octave>4</octave>
        </pitch>
        <duration>2</duration>
        <voice>1</voice>
        <type>eighth</type>
        <accidental>sharp</accidental>
        <stem>up</stem>
        <staff>1</staff>
        <beam number="1">continue</beam>
    </note>
    <direction placement="below">
        <direction-type>
            <wedge type="stop" number="1" relative-x="13.64"/>
        </direction-type>
        <staff>1</staff>
    </direction>
    <note default-x="176.87" default-y="-20.00">
        <pitch>
            <step>B</step>
            <octave>4</octave>
        </pitch>
        <duration>2</duration>
        <voice>1</voice>
        <type>eighth</type>
        <stem>up</stem>
        <staff>1</staff>
        <beam number="1">end</beam>
    </note>
    <backup>
        <duration>6</duration>
    </backup>
    <note default-x="107.53" default-y="-45.00">
        <pitch>
            <step>D</step>
            <octave>4</octave>
        </pitch>
        <duration>6</duration>
        <tie type="start"/>
        <voice>2</voice>
        <type>quarter</type>
        <dot/>
        <stem>down</stem>
        <staff>1</staff>
        <notations>
            <tied type="start"/>
        </notations>
    </note>
    <backup>
        <duration>12</duration>
    </backup>
    <note default-x="32.90" default-y="-115.00">
        <pitch>
            <step>C</step>
            <octave>4</octave>
        </pitch>
        <duration>2</duration>
        <voice>5</voice>
        <type>eighth</type>
        <stem>down</stem>
        <staff>2</staff>
        <beam number="1">begin</beam>
    </note>
    <note default-x="50.06" default-y="-125.00">
        <pitch>
            <step>A</step>
            <octave>3</octave>
        </pitch>
        <duration>2</duration>
        <voice>5</voice>
        <type>eighth</type>
        <stem>down</stem>
        <staff>2</staff>
        <beam number="1">continue</beam>
    </note>
    <note default-x="78.79" default-y="-135.00">
        <pitch>
            <step>F</step>
            <alter>1</alter>
            <octave>3</octave>
        </pitch>
        <duration>2</duration>
        <voice>5</voice>
        <type>eighth</type>
        <stem>down</stem>
        <staff>2</staff>
        <beam number="1">end</beam>
        <notations>
            <slur type="stop" number="2"/>
        </notations>
    </note>
    <note default-x="107.53" default-y="-130.00">
        <pitch>
            <step>G</step>
            <octave>3</octave>
        </pitch>
        <duration>2</duration>
        <voice>5</voice>
        <type>eighth</type>
        <stem>down</stem>
        <staff>2</staff>
        <beam number="1">begin</beam>
        <notations>
            <slur type="start" placement="above" number="2"/>
        </notations>
    </note>
    <note default-x="148.14" default-y="-135.00">
        <pitch>
            <step>F</step>
            <alter>1</alter>
            <octave>3</octave>
        </pitch>
        <duration>2</duration>
        <voice>5</voice>
        <type>eighth</type>
        <stem>down</stem>
        <staff>2</staff>
        <beam number="1">continue</beam>
    </note>
    <note default-x="176.87" default-y="-130.00">
        <pitch>
            <step>G</step>
            <octave>3</octave>
        </pitch>
        <duration>2</duration>
        <voice>5</voice>
        <type>eighth</type>
        <stem>down</stem>
        <staff>2</staff>
        <beam number="1">end</beam>
    </note>
    <backup>
        <duration>12</duration>
    </backup>
    <note default-x="12.00" default-y="-110.00">
        <pitch>
            <step>D</step>
            <octave>4</octave>
        </pitch>
        <duration>6</duration>
        <voice>6</voice>
        <type>quarter</type>
        <dot/>
        <stem>up</stem>
        <staff>2</staff>
    </note>
    <forward>
        <duration>6</duration>
    </forward>
</measure>
<measure number="32" width="197.66">
    <direction placement="below">
        <direction-type>
            <wedge type="diminuendo" number="1" default-y="-75.91" relative-x="0.64"/>
        </direction-type>
        <staff>1</staff>
    </direction>
    <note default-x="12.00" default-y="-15.00">
        <pitch>
            <step>C</step>
            <octave>5</octave>
        </pitch>
        <duration>2</duration>
        <voice>1</voice>
        <type>eighth</type>
        <stem>up</stem>
        <staff>1</staff>
        <beam number="1">begin</beam>
    </note>
    <note default-x="48.55" default-y="-25.00">
        <pitch>
            <step>A</step>
            <octave>4</octave>
        </pitch>
        <duration>2</duration>
        <voice>1</voice>
        <type>eighth</type>
        <accidental>natural</accidental>
        <stem>up</stem>
        <staff>1</staff>
        <beam number="1">continue</beam>
    </note>
    <direction placement="below">
        <direction-type>
            <wedge type="stop" number="1" relative-x="0.64"/>
        </direction-type>
        <staff>1</staff>
    </direction>
    <note default-x="77.29" default-y="-35.00">
        <pitch>
            <step>F</step>
            <alter>1</alter>
            <octave>4</octave>
        </pitch>
        <duration>2</duration>
        <voice>1</voice>
        <type>eighth</type>
        <stem>up</stem>
        <staff>1</staff>
        <beam number="1">end</beam>
        <notations>
            <slur type="stop" number="1"/>
        </notations>
    </note>
    <direction placement="below">
        <direction-type>
            <dynamics default-x="3.29" default-y="-80.00" relative-x="-4.85" relative-y="-0.18">
                <p/>
            </dynamics>
        </direction-type>
        <staff>1</staff>
        <sound dynamics="54.44"/>
    </direction>
    <note default-x="106.02" default-y="-10.00">
        <pitch>
            <step>D</step>
            <octave>5</octave>
        </pitch>
        <duration>2</duration>
        <voice>1</voice>
        <type>eighth</type>
        <stem>down</stem>
        <staff>1</staff>
        <beam number="1">begin</beam>
        <notations>
            <slur type="start" placement="above" number="1"/>
        </notations>
    </note>
    <note default-x="138.60" default-y="-15.00">
        <pitch>
            <step>C</step>
            <alter>1</alter>
            <octave>5</octave>
        </pitch>
        <duration>2</duration>
        <voice>1</voice>
        <type>eighth</type>
        <accidental>sharp</accidental>
        <stem>down</stem>
        <staff>1</staff>
        <beam number="1">continue</beam>
    </note>
    <note default-x="167.33" default-y="-10.00">
        <pitch>
            <step>D</step>
            <octave>5</octave>
        </pitch>
        <duration>2</duration>
        <voice>1</voice>
        <type>eighth</type>
        <stem>down</stem>
        <staff>1</staff>
        <beam number="1">end</beam>
        <notations>
            <slur type="stop" number="1"/>
        </notations>
    </note>
    <backup>
        <duration>12</duration>
    </backup>
    <note default-x="12.00" default-y="-45.00">
        <pitch>
            <step>D</step>
            <octave>4</octave>
        </pitch>
        <duration>6</duration>
        <tie type="stop"/>
        <voice>2</voice>
        <type>quarter</type>
        <dot/>
        <stem>down</stem>
        <staff>1</staff>
        <notations>
            <tied type="stop"/>
        </notations>
    </note>
    <backup>
        <duration>6</duration>
    </backup>
    <note default-x="12.00" default-y="-125.00">
        <pitch>
            <step>A</step>
            <octave>3</octave>
        </pitch>
        <duration>2</duration>
        <voice>5</voice>
        <type>eighth</type>
        <stem>down</stem>
        <staff>2</staff>
        <beam number="1">begin</beam>
    </note>
    <note default-x="48.55" default-y="-135.00">
        <pitch>
            <step>F</step>
            <alter>1</alter>
            <octave>3</octave>
        </pitch>
        <duration>2</duration>
        <voice>5</voice>
        <type>eighth</type>
        <stem>down</stem>
        <staff>2</staff>
        <beam number="1">continue</beam>
    </note>
    <note default-x="77.29" default-y="-145.00">
        <pitch>
            <step>D</step>
            <octave>3</octave>
        </pitch>
        <duration>2</duration>
        <voice>5</voice>
        <type>eighth</type>
        <stem>down</stem>
        <staff>2</staff>
        <beam number="1">end</beam>
        <notations>
            <slur type="stop" number="2"/>
        </notations>
    </note>
    <note default-x="106.02" default-y="-145.00">
        <pitch>
            <step>D</step>
            <octave>3</octave>
        </pitch>
        <duration>2</duration>
        <voice>5</voice>
        <type>eighth</type>
        <stem>down</stem>
        <staff>2</staff>
        <beam number="1">begin</beam>
        <notations>
            <slur type="start" number="1"/>
        </notations>
    </note>
    <note default-x="138.60" default-y="-140.00">
        <pitch>
            <step>E</step>
            <octave>3</octave>
        </pitch>
        <duration>2</duration>
        <voice>5</voice>
        <type>eighth</type>
        <stem>down</stem>
        <staff>2</staff>
        <beam number="1">continue</beam>
    </note>
    <note default-x="167.33" default-y="-135.00">
        <pitch>
            <step>F</step>
            <alter>1</alter>
            <octave>3</octave>
        </pitch>
        <duration>2</duration>
        <voice>5</voice>
        <type>eighth</type>
        <stem>down</stem>
        <staff>2</staff>
        <beam number="1">end</beam>
        <notations>
            <slur type="stop" number="1"/>
        </notations>
    </note>
    <backup>
        <duration>6</duration>
    </backup>
    <note default-x="106.02" default-y="-110.00">
        <pitch>
            <step>D</step>
            <octave>4</octave>
        </pitch>
        <duration>6</duration>
        <voice>6</voice>
        <type>quarter</type>
        <dot/>
        <stem>up</stem>
        <staff>2</staff>
    </note>
</measure>
```
</details>

As expected, MuseScore produces the following rendering:

![Expected Behavior](https://user-images.githubusercontent.com/13868083/95982235-2a847c00-0e20-11eb-9900-29932ea4f9c2.png)

However, (the demo of) OSMD produces the following rendering:

![Observed Behavior](https://user-images.githubusercontent.com/13868083/95982222-25bfc800-0e20-11eb-8cf6-b4173ffcbd91.png)

I would expect OSMD to render the dotted quarter notes as MuseScore does.

**Comment by sschmidTU** [comment]
the problem is probably the rare case of a dotted quarter starting on the second beat of the measure (with no rest note on the first beat in that voice) while the other voice is in triplets.
May be a Vexflow issue and in that case harder to fix, but it could also be fixable in OSMD.

**Comment by sschmidTU** [comment]
This has been fixed some time ago.
<img width="544" alt="image" src="https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/assets/33069673/0bc30c01-d927-4c96-827f-c20dd993be3f">

I think this was due to the fix of accepting multiple ghost notes (#1062), but not sure.
